src/compiler/builderPublic.ts

```diff
@@ -0,0 +1,160 @@
+namespace ts {

```

Comment:
x.ts/xPublic.ts is a fine naming scheme except that it makes bash-style completions worse because bu<TAB> doesn't produce a filename.

One solution is to use a public- prefix, which also seems bad. Or maybe a subdirectory. I don't really have *any* good ideas. Maybe this situation is temporary? I hope?

Reply:
I like @sandersn 's idea of subfolder for internal. If we are not making subfolder i would like having `xInternal` and `x` so the file names exposes at smaller scale are nicer and without public